### DISCUSSION TITLE
Indexing a Manifold Contract

### DISCUSSION THREAD

**Discussion by 0xKNNY** [f8n/fnd-protocol#113] [Q&A] [answered]
Is there a way for the end-user to index a manifold contract instead of asking Foundation to index it for them, and if so what would that process be

**Comment by HardlyDifficult** [comment, marked as answer]
The market contract works with any ERC721 NFT, including Manifold. So if you are integrating on a custom site - it's ready to use as is.

In terms of surfacing Manifold NFTs on foundation.app, we've pulled in a few by hand to help validate our systems work for that. We plan on opening it up so they all appear on our site soon.

**Reply by 0xKNNY** [reply]
Thank you for replying. Another question, is Foundation planning on opening the source code for the front-end of foundation.app to allow artists to easily fork their own marketplace utilizing the protocol

**Reply by HardlyDifficult** [reply]
We are working on an easy to integrate SDK and will include example frontend implementations that you will be able to fork to build from. This should give you everything you need to build a foundation.app like experience. For now at least, we don't have any near term plans to open source our full app implementation.
